templates/parachain/node/src/chain_spec.rs

```diff
@@ -1,6 +1,9 @@
 use parachain_template_runtime as runtime;
-use sc_chain_spec::{ChainSpecExtension, ChainSpecGroup};
-use sc_service::ChainType;
+use polkadot_sdk::{
```

Comment:
I think the pattern that @ggwpez envisioned here is to prepend the imports with a `use polkadot_sdk::*`? not sure if rustfmt then mixes them up.

Reply:
Just do `use polkadot_sdk::*` and keep the code as-is.  
For nested modules you have to import them again but otherwise it works.

Reply:
Thanks ! Done. Looks much better.